config/default_records/submission_templates/016_bge.wip.yml

```diff
@@ -0,0 +1,22 @@
+# Y24-366
+Limber-Htp - BGE PCR Free:
+  name: "Limber-Htp - GBE PCR Free"
+  submission_class_name: "LinearSubmission"
+  related_records:
+    product_line_name: Illumina-HTP
+    product_catalogue_name: PFHSqX
+    request_type_keys: ["limber_bge_pcr_free", "limber_multiplexing"]
+Limber-Htp - BGE ISC:
+  name: "Limber-Htp - BGE ISC "
+  submission_class_name: "AutomatedOrder"
+  related_records:
+    product_line_name: Illumina-HTP
+    product_catalogue_name: ISC
+    request_type_keys: ["limber_bge_isc", "limber_multiplexing", "illumina_htp_novaseqx_paired_end_sequencing"] # Other request types?
+Limber-Htp - BGE Transition - Automated:
+  name: "Limber-Htp - BGE Transition "
+  submission_class_name: "AutomatedOrder"
+  related_records:
+    product_line_name: Illumina-HTP
+    product_catalogue_name: ISC # Different product catalogue?
```

Comment:
Good question, think we probably need a new product catalogue for BGE which all these submissions should be a part of.